src/providers/details/tabs/Credentials/components/utils/getAuthTypeRequiredFields.ts

```diff
@@ -0,0 +1,56 @@
+import { OpenstackSecretFieldsId } from 'src/providers/utils/constants';
+
+import type { IoK8sApiCoreV1Secret } from '@kubev2v/types';
+
+export const getAuthTypeRequiredFields = (
+  secret: IoK8sApiCoreV1Secret,
+  authType: string,
+): OpenstackSecretFieldsId[] => {
+  switch (authType) {
+    case 'password':
```

Comment:
please use SecretFieldsId enum for case keys

Reply:
Here those const values refer to field values and not field names, so it will be wrong to use those consts.